transport/api/src/main/resources/transport-client-error-messages.properties

```diff
@@ -11,6 +11,9 @@
 #     Eurotech - initial API and implementation
 #
 ###############################################################################
-CLIENT_GET=Cannot get an instance of the transport client to connect to host: {0}
+CLIENT_GET=Cannot get an instance of the TransportClient to connect to host: {0}
+CLIENT_GET_WITH_CAUSE=Cannot get an instance of the TransportClient to connect to host: {0}. Caused by: {1}
+CLIENT_POOL_EXHAUSTED=Cannot get an instance of the TransportClient to connect to host {0} within the configured timeout of {1}ms
```

Comment:
Change this to
`Cannot connect to the Message Broker {0} within the configured timeout of {1}ms`